src/components/Form/Step5Confirm/Step5SummarySection.js

```diff
@@ -0,0 +1,278 @@
+              <td className="wmnds-text-align-right wmnds-p-r-none">
+                <ChangeLink changeStepTo={2} />
+              </td>
+            </tr>
+          )}
+        </tbody>
+      </table>
+
+      {(CustomerType !== 'Scratchcard' || CustomerType !== 'ClassPass') && (
+        <>
+          <div
+            className={`wmnds-m-b-sm wmnds-m-t-xl wmnds-grid wmnds-grid--justify-between ${style.serviceAdded}`}
+          >
+            <h3 className="wmnds-col-2-3">Ticket Details</h3>
+            <ChangeLink changeStepTo={3} />
+          </div>
+          <table className="wmnds-table wmnds-m-b-xl wmnds-table--without-header">
+            <tbody>
+              {DirectDebitNumber && (
+                <tr>
+                  <th scope="row">Direct Debit reference</th>
+                  <td className={`${style.tableColspan2}`}>
+                    {DirectDebitNumber === '0'
+                      ? `I can't find my Direct Debit reference`
+                      : DirectDebitNumber}
+                  </td>
+                </tr>
+              )}
+              {CardNumber && (
+                <tr>
+                  <th scope="row">Swift card number</th>
+                  <td className={`${style.tableColspan2}`}>{CardNumber}</td>
+                </tr>
+              )}
+              {TicketNumber && (
+                <tr>
+                  <th scope="row">Ticket number</th>
+                  <td className={`${style.tableColspan2}`}>{TicketNumber}</td>
+                </tr>
+              )}
+              {LastUsedDate && (
+                <tr>
+                  <th scope="row">Ticket last used</th>
+                  <td className={`${style.tableColspan2}`}>
+                    {getDateString(LastUsedDate)}
+                  </td>
+                </tr>
+              )}
+              {CovidTravel && (
+                <tr>
+                  <th scope="row">
+                    Did you stop using your ticket to travel because of the
+                    coronavirus (COVID-19) pandemic?
+                  </th>
+                  <td className={`${style.tableColspan2}`}>{CovidTravel}</td>
+                </tr>
+              )}
+              {PhotoBase64 && (
+                <tr>
+                  <th scope="row">Photo of ripped ticket</th>
+                  <td width="60%">
+                    {/* Render image from base64 (generated in file upload step) */}
+                    <img
+                      src={`data:image/${PhotoBase64Extension};base64, ${PhotoBase64}`}
+                      alt="Ticket upload"
+                    />
+                  </td>
+                </tr>
+              )}
+            </tbody>
+          </table>
+        </>
+      )}
+      <div
+        className={`wmnds-m-b-sm wmnds-m-t-xl wmnds-grid wmnds-grid--justify-between ${style.serviceAdded}`}
+      >
+        <h3 className="wmnds-col-2-3">Personal Details</h3>
+        <ChangeLink changeStepTo={4} />
```

Comment:
Some of these change links don't look too good:

![image](https://user-images.githubusercontent.com/13015201/101485076-87993b80-3952-11eb-8112-44a7da2994da.png)